client/my-sites/sidebar/index.jsx

```diff
@@ -493,6 +493,30 @@ export class MySitesSidebar extends Component {
 		);
 	}
 
+	trackHostingClick = () => {
+		this.trackMenuItemClick( 'hosting' );
+		this.onNavigate();
+	};
+
+	hosting() {
+		const { translate, path, site, isAtomicSite, siteSuffix } = this.props;
+
+		if ( ! site || ! isAtomicSite || ! isEnabled( 'hosting' ) ) {
```

Comment:
@gwwar when you get a chance, do you mind taking a look at this to make sure it won't be exposed? :) second-guessing slightly after almost merging it in 😅

Reply:
Haha, don't worry I've caught stuff last second on staging :)

Reply:
Testing instructions can include disabling the flag explicitly:

```jsx
DISABLE_FEATURES=hosting npm start
```

Reply:
Thanks so much!! Much appreciated :)